client/my-sites/customer-home/main.jsx

```diff
@@ -284,27 +293,24 @@ class Home extends Component {
 							</p>
 						</div>
 					</Card>
-				) }
-				{ ! siteIsUnlaunched && 'launched' === checklistMode ? (
-					<Card className="customer-home__launch-card" highlight="info">
+				);
+			case 'celebrate-site-setup-complete':
+				return (
+					<Card className="customer-home__site-setup-complete-card" highlight="info">
 						<img
 							src={ fireworksIllustration }
 							aria-hidden="true"
-							className="customer-home__launch-fireworks"
+							className="customer-home__site-setup-complete-fireworks"
 							alt=""
 						/>
-						<div className="customer-home__launch-card-text">
-							<CardHeading>{ translate( 'You launched your site!' ) }</CardHeading>
-							<p className="customer-home__launch-card-subtext">
-								{ this.getChecklistSubHeaderText() }
-							</p>
+						<div className="customer-home__site-setup-complete-card-text">
+							<CardHeading>
+								{ translate( "You've completed each item in your checklist" ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'You've completed each item in your checklist.' )` *ES Score: 12*